clang/tools/libclang/CIndexer.cpp

```diff
@@ -125,21 +126,28 @@ const std::string &CIndexer::getClangResourcesPath() {
 #elif defined(_AIX)
   getClangResourcesPathImplAIX(LibClangPath);
 #else
+  bool pathFound = false;
```

Comment:
nit: Variables should start with upper case

Reply:
thanks, done!